client/pdo/client/controller/commands/create.py

```diff
@@ -90,9 +90,19 @@ def __create_contract(ledger_config, client_keys, preferred_eservice_client, ese
 
     logger.info('Contract state created successfully')
 
-    logger.info('Saving the initial contract state in the ledger...')
-
-    cclinit_result = initialize_response.submit_initialize_transaction(ledger_config, wait=30.0)
+    # submit the commit task: (a commit task replicates change-set and submits the corresponding transaction)
+    try:
+        initialize_response.commit_asynchronously(ledger_config, wait=30, use_ledger=True)
```

Comment:
Is this 'wait' the timeout after which the async worker gives up?  If so, maybe timeout might be a more meaningful name?  Or is this a wait for getting into the queue to the worker?

If the former, how would a time-out manifest laterin wait_for_commit?

Reply:
the wait is the timeout after which the async worker (for submitting transactions) simply moves on to the next transaction task. this wait as part of the orignal transaction submitter. 

Thanks for raising this. I realize that wait does not perhaps make sense after the new commit module. it gives a wrong impression that the commit waits for 30 seconds, which is clearly not the case. will fix  this.

Reply:
could you add another issue to resolve "wait" in an async replication/commit?

Reply:
Sure, thanks, created issue 218.